### PR TITLE
Add bcftools annotate

### PR DESCRIPTION
**Pull request by grantn5**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -538,6 +538,18 @@
                     "help_text": "Sets the format of the output-file from VEP. Available formats: json, tab and vcf.",
                     "fa_icon": "fas fa-table",
                     "hidden": true
+                },
+                "annotations": {
```
Comment (FriederikeHanssen) [inline nextflow_schema.json:542]:
```suggestion
                "bcftools_annotations": {
```

File: subworkflows/local/vcf_annotate_all/main.nf [hunk 2]
```diff
@@ -26,6 +30,15 @@ workflow VCF_ANNOTATE_ALL {
     json_ann = Channel.empty()
     versions = Channel.empty()
 
+    vcf_ann.view()
```
Comment (FriederikeHanssen) [inline subworkflows/local/vcf_annotate_all/main.nf:33]:
```suggestion
```

**Comment by grantn5** [comment]
@FriederikeHanssen

File: nextflow_schema.json [hunk 3]
```diff
@@ -100,7 +100,7 @@
...
-                    "pattern": "^((ascat|cnvkit|controlfreec|deepvariant|freebayes|haplotypecaller|sentieon_dnascope|sentieon_haplotyper|manta|merge|mpileup|msisensorpro|mutect2|sentieon_dedup|snpeff|strelka|tiddit|vep)?,?)*(?<!,)$"
+                    "pattern": "^((ascat|bcfann|cnvkit|controlfreec|deepvariant|freebayes|haplotypecaller|sentieon_haplotyper|manta|merge|mpileup|msisensorpro|mutect2|sentieon_dedup|snpeff|strelka|tiddit|vep)?,?)*(?<!,)$"
```
Comment (maxulysse) [inline nextflow_schema.json:103]:
```suggestion
                    "pattern": "^((ascat|bcfann|cnvkit|controlfreec|deepvariant|freebayes|haplotypecaller|sentieon_dnascope|sentieon_haplotyper|manta|merge|mpileup|msisensorpro|mutect2|sentieon_dedup|snpeff|strelka|tiddit|vep)?,?)*(?<!,)$"
```

File: nextflow_schema.json [hunk 4]
```diff
@@ -1058,5 +1070,11 @@
...
             "$ref": "#/definitions/generic_options"
         }
-    ]
+    ],
+    "properties": {
+        "validationSchemaIgnoreParams": {
+            "type": "string",
+            "default": "cf_ploidy,genomes,igenomes_base"
+        }
+    }
```
Comment (maxulysse) [inline nextflow_schema.json:1079]: where is this coming from?

Comment (grantn5) [reply nextflow_schema.json:1079]: I have absolutely no Idea, I will remove

**Review by maxulysse — APPROVED** [review]
A couple of stuff that we could improve but not a problem for me

**Review by FriederikeHanssen — APPROVED** [review]

**Comment by FriederikeHanssen** [comment]
Taking back my approval :D we need a test

**Pull request merged**